src/iuse.cpp

```diff
@@ -835,7 +835,7 @@ int iuse::flu_vaccine( player *p, item *it, bool, const tripoint & )
 {
     p->add_msg_if_player( _( "You inject the vaccine." ) );
     p->add_msg_if_player( m_good, _( "You no longer need to fear the flu, at least for some time." ) );
-    p->add_effect( effect_flushot, 30_days, num_bp, true );
+    p->add_effect( effect_flushot, 6_months, num_bp, false );
```

Comment:
```suggestion
    p->add_effect( effect_flushot, 24_weeks, num_bp, false );
```

There isn't a `_months` currently.

Reply:
Thank you @ralreegorganon , this was a careless oversight by me.